samples/graph-change-notification-team-channel-nodejs/nodejs/client/src/components/channelnotification.jsx

```diff
@@ -0,0 +1,103 @@
+// <copyright file="dashboard.jsx" company="Microsoft Corporation">
```

Comment:
File name is different and here it's different.
Rename it as channel-notification.jsx